server/routes/api.js

```diff
@@ -21,4 +20,10 @@ router.use('/search', function(req, res) {
     });
 });
 
+router.use(function(req, res) {
+    res.status(404).json({
```

Comment:
Implement an error middleware for the API endpoints (just like the one in the web endpoints) and handle this on the middleware